stackit/internal/services/loadbalancer/loadbalancer/resource.go

```diff
@@ -446,6 +457,18 @@ The example below uses OpenStack to create the network, router, a public IP addr
 							Description: descriptions["target_port"],
 							Required:    true,
 						},
+						"session_persistence": schema.SingleNestedAttribute{
+							Description: descriptions["session_persistence"],
+							Optional:    true,
```

Comment:
In the LoadBalancer OAS seems like every field is required, just wanted to double check that this field is optional